src/app/cli/src/graphql.ml

```diff
@@ -84,6 +84,13 @@ module Types = struct
     (** Unix form of time, which is the number of milliseconds that elapsed from January 1, 1970 *)
     let date = Time.to_string
 
+    (** string representation of Trust_system.Banned_status *)
+    let banned_status = function
+      | Trust_system.Banned_status.Unbanned ->
+          None
+      | Banned_until tm ->
+          Some (Time.to_string_abs ~zone:Time.Zone.utc tm)
```

Comment:
can you do `Some (date tm)` instead?

Reply:
You mean something like, `Stringable.time tm` ?